### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by SirishaKonchada**
Basics

### CODE REVIEW

File: A.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ssh remote_user_name@IP_address
+hostname
+service service_name start/stop/restart
+sudo get-apt install package_name
```
Comment (dileepsai21) [inline A.txt:4]: seems this command is wrong... please correct this

Comment (SirishaKonchada) [reply A.txt:4]: which one?

File: A.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ssh remote_user_name@IP_address
+hostname
```
Comment (dileepsai21) [inline A.txt:2]: this is correct

File: A.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+ssh remote_user_name@IP_address
```
Comment (dileepsai21) [inline A.txt:1]: example: ssh user@10.30.30.112

File: A.txt [hunk 4]
```diff
@@ -0,0 +1,4 @@
+ssh remote_user_name@IP_address
+hostname
+service service_name start/stop/restart
```
Comment (dileepsai21) [inline A.txt:3]: example: sudo service apache2 start

Comment (SirishaKonchada) [reply A.txt:3]: acha okay

**Pull request merged**